connector/oidc/oidc.go

```diff
@@ -76,7 +76,7 @@ type Config struct {
 	UserNameKey string `json:"userNameKey"`
 
 	// PromptType will be used fot the prompt parameter (when offline_access, by default prompt=consent)
-	PromptType string `json:"promptType"`
+	PromptType *string `json:"promptType"`
```

Comment:
If this is a deviation from the spec (even if due to some IdP implementing it improperly) we should clearly state that both in the code and documentation.

Reply:
It doesn't seem like an absolute deviation. This is not fixed anywhere that IdP must ask for consent if the prompt is empty by default. The RFC only says how to behave if the prompt is provided.

For me, it seems like consent is the Dex default only (which we state in the doc and the code).

Reply:
SGTM